plugins/firebase_lumberdash/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
     sdk: flutter
   lumberdash: ^2.1.0
   meta: ^1.0.0
-  firebase_analytics: ^5.0.2
+  firebase_analytics: ^6.0.0
```

Comment:
Thank you for the contribution, but we ended up updating the dependency with https://github.com/bmw-tech/lumberdash/pull/58
Closing this issue.